modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,133 @@ endif::[]
+
+=== Mule v3.x
+You can download the OAuth Provider from Exchange.
+
+See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-external-oauth2-provider/[External OAuth 2.0 server for Anypoint Platform] asset in Exchange to download it.
+
+=== Mule v4.x
+This feature is available from Mule v4.2.0.
+
+See the https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset in Exchange to download it.
+
+== OAuth 2.0 dance
+
+The authentication process performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This process is called the xref:oauth-dance-about.adoc[OAuth 2.0 dance].
+
+Mule OAuth 2.0 provider supports all xref:oauth-grant-types-about.adoc[grant types].
+
+The entity that requests access to a resource protected by Mule OAuth 2.0 access token enforcement policy is a `Client`.
+
+=== Endpoints
+
+image::mule-oauth-provider.png[height=150,width=250]
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Purpose | Default path | Description
+| Token validation
+| /validate
+| Used to verify the validity of the token (eg. it lets you determine if the token is expired, revoked or fake). You may define more than one validation endpoint, each of them enforcing different scopes. See <<Scopes>> for more information about this.
+| Authorization
+| /authorize
+| Used to obtain a token when the client is running in a browser using a scripting language or the client is a web server.
+| Access
+| /access_token
+| Used when the client is owned by the same entity as the Mule OAuth 2.0 provider or when the client himself is the same entity that owns the protected resource.
+| Token revocation
+| /revoke
+| [Optional] Used to revoke a valid token, so it becomes invalid.
+|===
+
+=== Scopes
+
+OAuth 2.0 https://tools.ietf.org/html/rfc6749#page-23[scopes] are a way to further limit access to a resource protected by OAuth. You may define words like `READ`, `WRITE`, or some other words that make sense in the context of your organization (eg. `CONTRACTOR`, `PUBLIC`, `EMPLOYEES_ONLY`, etc).
+
+You can define scopes in three different places:
+
+* In the universal set definition, `defaultScopes`, where you define all possible scopes.
+  Example:
++
+[source,xml,linenums]
+----
+<oauth2-provider:config
+  scopes="CONTRACTOR, PUBLIC_READ, EMPLOYEES_ONLY, WRITE"
+  >
+  ...
+</oauth2-provider:config>
+----
+* In the validation endpoint, `/validate`, to enforce a particular subset of the previously defined scopes.
+* In the `Mule OAuth 2.0 access token enforcement` policy, where you can enforce scopes by specifying a space-separated list.
+
+== Example fragment using Mule v4.2.x or later
+
+[source,xml,linenums]
```

Comment:
for more details check the link `blah blah ` (or say it can be found below)

Reply:
Done.